pr_testing/test_multiple_prs.sh

```diff
@@ -703,12 +703,12 @@ if [ "X$DO_STATIC_CHECKS" = "Xtrue" -a "X$CMSSW_PR" != X -a "$RUN_TESTS" = "true
     echo '--------------------------------------'
     echo "Changed files:"
     echo ""
-    cat $WORKSPACE/changed-files
+    curl -s -L https://patch-diff.githubusercontent.com/raw/${REPOSITORY}/pull/${PULL_REQUEST}.patch | grep '^diff --git ' | sed 's|.* a/||;s|  *b/.*||' | sort | uniq > $WORKSPACE/all-changed-files.txt
```

Comment:
you need to use `$PR_REPO` and `$PR_NUM` in this script

Reply:
PR_NUM is a md5 hash. PR_NUMBER appears to be the correct var.